automation/infrastructure/terraform/kcs/install_kubeflow/monitoring.env.yaml

```diff
@@ -0,0 +1,81 @@
+namespaceOverride: "kubeflow-monitoring"
+
+nodeExporter:
+  enabled: false
+kubeStateMetrics:
+  enabled: false
+
+prometheusOperator:
+  serviceMonitorSelectorNilUsesHelmValues: false
+  namespaces:
+    releaseNamespace: true
+    additional:
+    - kubeflow
+
+prometheus:
+  prometheusSpec:
+    additionalScrapeConfigs:
+      # Custom monitoring: monitoring ml-pipeline (the Kubeflow Pipelines API server)
+      - job_name: 'ml-pipeline'
+        scrape_interval: 15s
+        static_configs:
+          - targets: ['ml-pipeline.kubeflow:8888']
+
+      - job_name: 'kubeflow-pods'
+        scrape_interval: 15s
+        authorization:
+          credentials_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+        tls_config:
+          ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+          insecure_skip_verify: true
+        kubernetes_sd_configs:
+          - role: pod
```

Comment:
why do we need to do it on pods? are services not enough?

Reply:
I am not sure about user's namespaces. They can create monitorable pods, not services.